docs/nidigital/class.rst

```diff
@@ -3384,6 +3384,18 @@ channels
         passes a string of :python:`'0, 1, 2'` to the set attribute function.
 
 
+pins
+----
+
+    .. py:attribute:: nidigital.Session.pins[]
+
+        .. code:: python
+
+            session.pins['0-2'].channel_enabled = True
```

Comment:
This is not a good example. How difficult is it to do something `nidigital` specific?
#1125 created